pkg/hostagent/hostagent.go

```diff
@@ -568,13 +568,19 @@ func (a *HostAgent) watchGuestAgentEvents(ctx context.Context) {
 		guestSocketAddr = fmt.Sprintf("0.0.0.0:%d", a.vSockPort)
 	}
 
+	go func() {
+		err := a.startInotify(ctx, guestSocketAddr, localUnix, remoteUnix)
```

Comment:
This should be enabled only with `limactl start --mount-inotify`, and it should be marked experimental.

Reply:
Should we expose this via yaml or just via cmd ?

Reply:
Yaml, just like other mount options

Reply:
Done